pluma/stream/zeromq.py

```diff
@@ -206,4 +206,48 @@ def __init__(self, eventcode: int, **kw):
                  ('Subject.Position.Y', np.single),
                  ('Point.Position.X', np.single),
                  ('Point.Position.Y', np.single)]],
+            **kw)
+class UnityNewSceneStream(ZmqStream):
```

Comment:
```suggestion
class ProtocolNewSceneStream(ZmqStream):
```

If we adopt the folder structure mentioned in https://github.com/emotional-cities/pluma-experiments/issues/89 we should probably rename the relevant stream classes to have a `Protocol` prefix for consistency.

Reply:
Solved in 2a7cc0b